docs/se/guides/07_db_client.adoc

```diff
@@ -0,0 +1,634 @@
+
+[source,java]
+.Add deleteBook to LibraryService:
+----
+private void deleteBook(ServerRequest serverRequest, ServerResponse serverResponse) {
+    String bookName = serverRequest.path().param("name");
+
+    dbClient.execute(exec ->  exec.namedDelete("delete-book", bookName))     // <1>
+                        .thenAccept(count -> serverResponse.status(Http.Status.NO_CONTENT_204).send())   // <2>
+                        .exceptionally(serverResponse::send);
+}
+----
+<1> Execute SQL script from application.yaml to remove a book from the library by its name.
+<2> The required book was removed, so a HTTP 204 is sent.
+
+To remove a book from the library, use the "delete-book" script in the way than previously. If the book is removed
+successfully, a HTTP 204 is sent back.
+
+== Build and Run the Library application
+
+The application is ready to be built and ran.
+
+[source,bash]
+.Run the following to build the application:
+----
+mvn package
+----
+
+Note that the tests are passing as the GreetService process was not modified. During this guide, we only added independent
+new content to the existing application.
+Make sure H2 is running and start the helidon quickstart by this command:
+
+[source,bash]
+.Run the application
+----
+java -jar target/helidon-quickstart-se.jar
+----
+
+Once the application starts, check the table LIBRARY is created in the H2 database. To do so, go to the
+H2 Server console and LIBRARY table should be present in the left column under `jdbc:h2:tcp://localhost:9092/~/test`.
+If it is not, try to refresh the page, and it should appear.
+
+Use `curl` to send request to the application:
+
+[source,bash]
+.Get a book from the library
+----
+curl -i http://localhost:8080/library/SomeBook
+
+HTTP/1.1 404 Not Found
+Date: Tue, 12 Jan 2021 14:00:48 +0100
+transfer-encoding: chunked
+connection: keep-alive
+----
+
+There is currently no book inside the library, so the application return a 404. Yet the application created an empty
+library table. Try to add a new book.
+
+[source,bash]
+.Add a book from the library
+----
+curl -i -X PUT -d "Fantasy" http://localhost:8080/library/HarryPotter
+
+HTTP/1.1 201 Created
+Date: Tue, 12 Jan 2021 14:01:08 +0100
+transfer-encoding: chunked
+connection: keep-alive
+----
+
+This command creates an HTTP PUT request with the genre `Fantasy` content at the address
+http://localhost:8080/library/{book-name}. The 201 code means that Harry Potter book was successfully added to
+the library. You can now try to get it !
+
+[source,bash]
+.Get Harry Potter from the library
+----
+curl -i http://localhost:8080/library/HarryPotter
+
+HTTP/1.1 200 OK
+Content-Type: text/plain
+Date: Tue, 12 Jan 2021 14:01:14 +0100
+connection: keep-alive
+content-length: 6
+
+Fantasy
+----
+
+The application accepted the request and return an HTTP 200 OK with the book genre which was added earlier.
```

Comment:
The application accepted the request and returned an HTTP 200 OK with the book genre that was added earlier.